crates/bevy_math/src/aspect_ratio.rs

```diff
@@ -10,18 +10,74 @@ use bevy_reflect::Reflect;
 #[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq))]
 pub struct AspectRatio(f32);
 
+#[derive(Debug, PartialEq)]
+pub enum AspectRatioError {
+    Zero,
```

Comment:
Each enum value will need its documentation string as well, to describe the error case. For example: 
```suggestion
    /// Error due to width or height having zero as a value.
    Zero,
```

Reply:
thx!